src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java

```diff
@@ -0,0 +1,126 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
Oh... unless you copy/pasted it from another pre-existing provider... in that case the header would be right as is